_posts/2023-11-09-in-progress-feature-on-openshift.adoc

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: 'Trying Out An In-progress Feature for WildFly on OpenShift'
+date: 2023-11-09
+tags: oidc authentication OpenShift
+synopsis: How to deploy a WildFly app secured with OIDC on OpenShift using an in-progress feature. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+A future release of WildFly will include the ability to add additional scope values when securing applications using
+OpenID Connect. This blog post demonstrates how to secure a WildFly application using OpenID Connect and configure scope
+values to request additional claims using an in-progress version of the feature. For updates on the status of the feature, 
+please keep an eye on https://issues.redhat.com/browse/WFLY-16532[WFLY-16532] and https://issues.redhat.com/browse/ELY-2574[ELY-2574]. 
```

Comment:
Because the availability of this feature in WildFly can be checked from the WFLY issue, I think we can just link to that one (and the ELY issue is also linked from there).

Reply:
updated! 
Thanks :)